pkg/cmd/sync/sync.go

```diff
@@ -115,9 +119,9 @@ func (o *SyncOptions) Run() error {
 		flag, err := kube.IsDaemonSetExists(client, "ksync", "kube-system")
 		if !flag || err != nil {
 			log.Logger().Infof("Initialising ksync")
-			// Deal with https://github.com/vapor-ware/ksync/issues/218
+			// Deal with https://github.com/ksync/ksync/issues/218
 			err = o.RunCommandInteractive(true, "ksync", "init", "--upgrade", "--image",
```

Comment:
Should that not go somehow through the Ksyncer interface?

Reply:
Yup, and it does now :)